### ISSUE TITLE
'go get' is no longer supported outside a module.

### ISSUE THREAD

**Issue by jelabarre59** [odeke-em/drive#1144]
The instructions on the main page are no longer valid for current versions of Go.  

The command "go get -u github.com/odeke-em/drive/cmd/drive"
should now be "go install github.com/odeke-em/drive/cmd/drive@latest"

**Comment by osalbahr** [comment]
I had the same issue

```
$ go get -u github.com/odeke-em/drive/cmd/drive
go: go.mod file not found in current directory or any parent directory.
	'go get' is no longer supported outside a module.
	To build and install a command, use 'go install' with a version,
	like 'go install example.com/cmd@latest'
	For more information, see https://golang.org/doc/go-get-install-deprecation
	or run 'go help get' or 'go help install'.
```

**Comment by fontseca** [comment]
```go install``` worked for me

**Comment by osalbahr** [comment]
It’s been a while, I forgot I even commented here lol. Is there an good alternative? This project seems dead, sadly.

**Comment by fontseca** [comment]
Yeah, it's sort of dead. I switched to [rclone for Google Drive](https://rclone.org/drive/). It meets my necessities.

**Comment by amhendley** [comment]
If anyone still needs this, just swap out the `go get` for `go install github.com/odeke-em/drive/cmd/drive@latest`

**Comment by jelabarre59** [comment]
I've ended up going with @fontseca's solution.  RClone can do exports of GoogleDocs to OpenDocument and PDF files as part of it's backup, so that does what I previously used "Drive' for.
